### ISSUE TITLE
👤 [User Story]: Fully automated IAC (including db setup)

### ISSUE THREAD

**Issue by edwardchalstrey1** [Seshat-Global-History-Databank/seshat#48]
### Role

Developer

### Desired Feature

Pulumi deployment includes db setup

### Benefit

Website can be delployed more quickly

### Acceptance Criteria

**GIVEN** The Pulumi script
**WHEN** Running `pulumi up`
**THEN** The Seshat site can be deployed to the cloud without [manual steps](https://seshat-global-history-databank.github.io/seshat/getting-started/setup/cloud/pulumi.html#manual-steps) related to database setup needing to be performed

### Dependencies

- [ ] #13 
- [ ] #46 
- [ ] #47

### Technical Notes

1. Need a way to upload a database dumpfile to the VM after it's created, but before the database is created
2. A way to set up the postgres user password
3. A way to add the new IP address to the `ALLOWED_HOSTS` in the django settings, which would need to be done before django is run

See discussion in #69 
See original issue: https://github.com/MajidBenam/seshat/issues/4

### Definition of Done

- [ ] The feature fulfills all acceptance criteria.
- [ ] The code is reviewed and meets team coding standards.
- [ ] All unit, integration, and end-to-end tests are passed.
- [ ] The feature meets accessibility standards set for the project.
- [ ] The code passes the CI/CD pipeline quality controls (style and linting).
- [ ] The feature is merged into the main branch.
- [ ] The feature is deployed and functional in the QA environment.
- [ ] The feature was demonstrated to the team and accepted by the product owner.
